tests/__sf_section_host.test.fish

```diff
@@ -0,0 +1,130 @@
+source $DIRNAME/spacefish_test_setup.fish
+
+function setup
+	spacefish_test_setup
+end
+
+
+test "Correctly shows hostname upon SSH connection"
+	(
+		set SSH_CONNECTION "192.168.0.100 12345 192.168.0.101 22"
+
+		set_color --bold fff
+		echo -n "at "
+		set_color normal
+		set_color --bold green
+		echo -n (hostname)
+		set_color normal
+		set_color --bold fff
+		echo -n " "
+		set_color normal
+		
+	) = (__sf_section_host)
+end
+
+test "Display user when SPACEFISH_HOST_SHOW is set to 'always'"
+	(
+		set SPACEFISH_HOST_SHOW always
+		set --erase SSH_CONNECTION
```

Comment:
To keep the tests atomic, I would suggest putting this line on a new line beneath line 21 as a cleanup step to the last test. 😀

Reply:
Oh, I tried this and it didn't work since it prematurely unset the SSH_CONNECTION. I think instead of going on line 21 (Green arrow) it has to go after line 22 (Red arrow), since it has to unset SSH *after* the comparison is made.

I don't know why silly me put it in the next test instead.

![image](https://user-images.githubusercontent.com/26250962/43237384-7d9dbcae-9081-11e8-9481-c3030dcf91c5.png)